textdb/textdb-gui/js/onstart.js

```diff
@@ -0,0 +1,133 @@
+/*
+	Gui <---> Flowchart.js Communication
+	GUIJSON <---> TEXTDBJSON Conversion
+	
+	@Author: Jimmy Wang
+*/
+
+// main function
+var setup = function(){
+	var data = {};
+
+	// Apply the plugin on a standard, empty div...
+	$('#the-flowchart').flowchart({
+		data: data
+	});
+	
+	var operatorI = 0;
+	
+	var defaultRegex = "zika\s*(virus|fever)";
```

Comment:
Constants should be named in upper case, and delimited by underscores. Like this -> `DEFAULT_REGEX`